setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name="docker_xylem",
-    version='0.1.4',
+    version='0.1.5',
```

Comment:
I think this needs to be `0.2.0` or something instead. It's a pretty big change.